osf/migrations/0003_aggregated_runsql_calls.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0002_adminlogentry'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            [
+                """
+                CREATE UNIQUE INDEX one_quickfiles_per_user ON public.osf_abstractnode USING btree (creator_id, type, is_deleted) WHERE (((type)::text = 'osf.quickfilesnode'::text) AND (is_deleted = false));
+                CREATE INDEX osf_abstractnode_collection_pub_del_type_index ON public.osf_abstractnode USING btree (is_public, is_deleted, type) WHERE ((is_public = true) AND (is_deleted = false) AND ((type)::text = 'osf.collection'::text));
+                CREATE INDEX osf_abstractnode_date_modified_ef1e2ad8 ON public.osf_abstractnode USING btree (last_logged);
+                CREATE INDEX osf_abstractnode_node_pub_del_type_index ON public.osf_abstractnode USING btree (is_public, is_deleted, type) WHERE ((is_public = true) AND (is_deleted = false) AND ((type)::text = 'osf.node'::text));
+                CREATE INDEX osf_abstractnode_registered_date_index ON public.osf_abstractnode USING btree (registered_date DESC);
+                CREATE INDEX osf_abstractnode_registration_pub_del_type_index ON public.osf_abstractnode USING btree (is_public, is_deleted, type) WHERE ((is_public = true) AND (is_deleted = false) AND ((type)::text = 'osf.registration'::text));
+                CREATE INDEX fileversion_date_created_desc ON public.osf_fileversion USING btree (created DESC);
+                CREATE INDEX fileversion_metadata_sha_arch_vault_index ON public.osf_fileversion USING btree (((metadata -> 'sha256'::text)), ((metadata -> 'archive'::text)), ((metadata -> 'vault'::text)));
+                CREATE INDEX nodelog__node_id_date_desc ON public.osf_nodelog USING btree (node_id, date DESC);
+                CREATE INDEX osf_nodelog_should_hide_nid ON public.osf_nodelog USING btree (should_hide, node_id);
+                CREATE UNIQUE INDEX osf_noderequest_target_creator_non_accepted ON public.osf_noderequest USING btree (target_id, creator_id) WHERE ((machine_state)::text <> 'accepted'::text);
+                CREATE INDEX lowercase_tag_index ON public.osf_tag USING btree (lower((name)::text), system);
```

Comment:
### More on indexes update

On `prod` server, there are 922 indexes
In `local` env on `develop`, there are 944 - 21 (silk) = 923 indexes
In `local` env on `feature/django_upgrade`, there are 945 - 21 (silk) = 924 indexes
In `local` evn on `feature/clean_squash` (this PR), there are 933 - 21 (silk) + 12 (new indexes added)= 924 indexes

First, we've verified the extra index between `local-develop` and `prod` is an existing "bug". This is trivial and can be neglected for now.

> `Institution` inherits `Loggable`, but that 0069 migration didn't actually run, any my script didn't manually create the index [here](https://github.com/CenterForOpenScience/osf.io/pull/7499/files)

```
osf_institution
osf_institution_last_logged_9ac963e2
CREATE INDEX osf_institution_last_logged_9ac963e2 ON public.osf_institution USING btree (last_logged)
```
Second, the one extra index is not introduced by the squash but by [one](https://github.com/CenterForOpenScience/osf.io/commit/4a8bd614f2c79c390c5acece38a24de2fd196653) of our post-migrate signal PRs. This probably doesn't matter though we no longer use `expires`. There maybe some performance hit due to indexing this one but not a blocker for `staging3` merge since its already there.

```
osf_cache_table
osf_cache_table_expires
CREATE INDEX osf_cache_table_expires ON public.osf_cache_table USING btree (expires)
```